packages/react-scripts/scripts/start.js

```diff
@@ -258,6 +259,31 @@ function run(port) {
   runDevServer(host, port, protocol);
 }
 
+function isProcessAReactApp(processCommand) {
+  return /^node .*react-scripts\/scripts\/start\.js\s?$/.test(processCommand);
+}
+
+function getProcessForPort(port) {
```

Comment:
Ya, didn't even notice that dir. Great! It allowed me to create some nice helpers too. 85e8151